t/40tls13-early-data.t

```diff
@@ -60,15 +60,17 @@ subtest "http/2" => sub {
             $hpack_hdr->(":authority", "127.0.0.1"),
             $hpack_hdr->(":path", $path),
         );
-        syswrite $fh, join '', (
+        syswrite $child_in, join '', (
             "PRI * HTTP/2.0\r\n\r\nSM\r\n\r\n",                             # preface
             "\x00\x00\x00\x04\x00\x00\x00\x00\x00",                         # SETTINGS
             "\x00\x00@{[chr length $hpack]}\x01\x05\x00\x00\x00\x01$hpack", # HEADERS
         );
-        # do not wait for idle-timeout
-        sleep 3;
-        kill 'KILL', $pid;
-        open $fh, "<", "$tempdir/resp.txt"
+
+        while (waitpid($pid, 0) != $pid) {
+            sleep 0.01;
```

Comment:
I do not think we need `sleep` here, because waitpid with the second argument _not_ set to `WNOHANG` is a blocking system call?

Reply:
oh, so why did you suggest while 53b5b28a730a8254436a76859c9ec0fe4ee6c651?

Reply:
Waitpid is a blocking system call (unless WNOHANG is being used). Most system calls, including waitpid, might return early, regardless of them being blocking or not. See documentation around `EINTR`.

Reply:
Ah, `EINTER`! Got it. Thank you.